typed-racket-lib/typed-racket/private/cast-table.rkt

```diff
@@ -16,10 +16,12 @@
 
 (define cast-table (make-free-id-table))
 
-;; cast-table-set! : Id Type-Stx -> Void
-(define (cast-table-set! id type-stx)
-  (free-id-table-set! cast-table id type-stx))
+;; cast-table-add! : Id Type-Stx -> Void
+(define (cast-table-add! id type-stx)
```

Comment:
Could we add a brief purpose statement to cast-table-add! -- i.e. why is there a list of casts that we're adding to? When would there be more than one vs just one? (so the function is a little more self documenting in what it does in the grand scheme of things)

Reply:
Okay. Are the new comments clear enough?